docs/index.asciidoc

```diff
@@ -335,11 +336,17 @@ If you set this you must also set the `password` option.
   * Value type is <<number,number>>
   * Default value is `200`
 
-How long to wait before checking if the connection is stale before executing a request on a connection using keepalive.
-# You may want to set this lower, possibly to 0 if you get connection errors regularly
-Quoting the Apache commons docs (this client is based Apache Commmons):
-'Defines period of inactivity in milliseconds after which persistent connections must be re-validated prior to being leased to the consumer. Non-positive value passed to this method disables connection validation. This check helps detect connections that have become stale (half-closed) while kept inactive in the pool.'
-See https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity(int)[these docs for more info]
+How long to wait before checking for a stale connection before executing a request on a connection using keepalive.
```

Comment:
Still needs some clarification. Does this set how long to wait before checking for a stale connection in order to determine if a keepalive request is needed?

Reply:
> Does this set how long to wait before checking for a stale connection in order to determine if a keepalive request is needed?

That is it, after an X amount of inactivity time a check is done.